scripts/o3de/ExportScripts/export_source_ios_xcode.py

```diff
@@ -0,0 +1,122 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import logging
+import sys
+
+import o3de.export_project as exp
+from o3de.export_project import process_command
+import pathlib
+
+def export_ios_xcode_project(ctx: exp.O3DEScriptExportContext,
+                             tools_build_folder: pathlib.Path,
+                             ios_build_folder: pathlib.Path,
+                             should_build_tools:bool = True,
+                             should_process_assets: bool = True,
+                             logger: logging.Logger|None = None):
+    """
+    This function serves as an initial exporter for project to the iOS platform. The steps in this code will generate
+    an XCode project file containing all necessary build information to produce iOS executables. 
```

Comment:
FYI: `Xcode` is the proper casing, not `XCode` (https://developer.apple.com/xcode/) . Can you change it for the entire file?